examples/with-supertokens/app/components/getServerSideSession.tsx

```diff
@@ -0,0 +1,90 @@
+import { cookies, headers } from 'next/headers';
+import { NextRequest, NextResponse } from 'next/server';
+import Session, { SessionContainer } from 'supertokens-node/recipe/session';
+import { ensureSuperTokensInit } from '../../config/backendConfig';
+
+ensureSuperTokensInit();
+
+export async function getServerSideSession(req?: NextRequest): Promise<{
```

Comment:
I see this function is mainly useful when using in SSR. Maybe we should rename it to getSSRSession

Reply:
We also use it in API route handlers.

Reply:
I don't think we should recommend to use this in API route, since we already have withSession